release-notes-3.1.1.md

```diff
@@ -0,0 +1,55 @@
+# Release Notes
+
+While the 3.1.1 release makes no mechanical changes to the OpenAPI 3.1.0 specification, it does introduce a number of notable improvements, including:
```

Comment:
"mechanical changes" feels a bit odd to me.  I'd use "changes to requirements" or "changes to mandatory requirements" (but I don't feel strongly about this).